.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+name: Main
+on:
+  push:
+  pull_request:
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install -r requirements.txt
+    - name: Freeze HTML
+      run: python fedoralovespython.py freeze
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    - if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
I'm a bit worried that this will deploy pull requests from `master` branches of random forks. 
Shouldn't this also check [`github.event_name`](https://docs.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context)?

Reply:
This works as expected in https://github.com/pyvec/python.cz/pull/436

The ref is `refs/remotes/pull/436/merge` on a PR.